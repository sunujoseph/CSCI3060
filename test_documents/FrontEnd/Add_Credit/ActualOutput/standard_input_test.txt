path recieved: C:\Users\pwner\Documents\Uoit\Year 5 Semester 2\CSCI 3060U\Project\CSCI3060\test_documents\FrontEnd\Add_Credit\ActualDTF\standard_input_test.txt
Enter Username: temp: |
addcreditÍÍÍÍÍÍÍıııı|
username: |userFS
addcredi| not found
Error: Not Logged In
Error: Not Logged In
Error: Not Logged In
