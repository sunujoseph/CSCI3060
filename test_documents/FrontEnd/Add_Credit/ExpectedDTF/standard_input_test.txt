06 userFS          FS 111211.11
00 userFS          FS 111211.11