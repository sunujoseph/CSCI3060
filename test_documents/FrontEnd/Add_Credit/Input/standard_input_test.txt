login
userFS
addcredit
100
logout
