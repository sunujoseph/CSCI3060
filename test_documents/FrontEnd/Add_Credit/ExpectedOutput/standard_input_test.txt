Enter user name: 
Enter amount of money to add to your account: 